server/models/database.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  last_name VARCHAR(50) NOT NULL,
+  first_name VARCHAR(50) NOT NULL,
+  email VARCHAR(255) UNIQUE NOT NULL,
+  password VARCHAR(50) NOT NULL,
```

Comment:
password length should be the size of the hash function you use. (sha256 => 256bits, 8 bytes) I am not sure if sha256 is still a suggested hash algo.

Reply:
Also look up adding a salt.

Reply:
Actually, there is no `BINARY` keyword in Postgres so I will just use the other option, `CHAR(64)`.

Extra info:
- There exists a keyword BYTEA in Postgres, but the documentation is not very helpful and everyone just uses CHAR(64) with the hash stored in hex anyways.
- Postgres has an extension called pgcrypto that can handle the password salt, hashing, and storage. (I will not implement this since the account management will be handled by AWS Cognito.)